frontend/libs/ui-modules/src/modules/segments/hooks/index.tsx

```
@@ -25,29 +29,83 @@ export const getFieldsProperties = (propertyType?: string) => {
+    sessionKey: 'property_cursor',
+  });
+  const { data, loading, fetchMore } = useQuery(query, {
+    variables: {
+      limit: PER_PAGE,
+      cursor,
+      searchValue: searchValue ?? undefined,
+    },
+    skip,
+  });
+
+  const { list, totalCount, pageInfo } = (data || {})[queryName] || {};
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)